addon/templates/components/head-layout.hbs

```diff
@@ -1 +1,3 @@
-<meta name="ember-cli-head-start">{{head-content}}<meta name="ember-cli-head-end">
+{{#-in-element headElement}}
```

Comment:
FYI - We have started taking steps to make this public API from the Glimmer VM perspective (see https://github.com/glimmerjs/glimmer-vm/pull/619). It is unclear what Ember version that PR will correspond to, but it is a good step forward for our use case (and shows a future where we can drop the private `-` prefix).